.eslintrc.js

```diff
@@ -17,11 +18,14 @@ module.exports = {
     ecmaVersion: 2018,
     sourceType: "module"
   },
-  plugins: ["@typescript-eslint"],
```

Comment:
included tests files now so added the eslint jest plugin. Highlighted a couple of nice improvements.